readme.dev.txt

```diff
@@ -14,4 +14,8 @@ Start the containers
 "docker-compose start"
 
 Stop the containers
-"docker-compose stop"
\ No newline at end of file
+"docker-compose stop"
+
+Build the plugin:
+"composer install"
+"npm install"
```

Comment:
Change to `npm ci`. This will prevent us from accidentally updating any packages unless we need to.

Reply:
Also, this readme still references the docker container that we used to have in our plugins. Maybe that should be removed since its no longer in the code?